blockchain/blockchain.go

```diff
@@ -1796,6 +1797,12 @@ func (bc *BlockChain) insertChain(chain types.Blocks) (int, []interface{}, []*ty
 				// current block is not the last one, so prefetch the right next block
 				followup := chain[i+1]
 				go func(start time.Time) {
+					defer func() {
+						if err := recover(); err != nil {
+							logger.Error("Got panic and recovered from prefetcher", "err", err)
+						}
+					}()
+
```

Comment:
Does this recovering logic necessary to `txPrefetcher` as well? on blockchain.go#350

Reply:
It might, but I am not sure yet. tx prefechers process the inserting block and it should not have processing errors. 
I will consider it later in another PR if needed

Reply:
Alright. Thanks :)